5.0/en/0x05-For-Users-Of-4.0.md

```diff
@@ -1,86 +1,84 @@
+Feedback indicated that the large number of Level 1 requirements (~120), combined with its designation as the "minimum" level, discouraged adoption. Organizations often either abandoned the standard or implemented only a subset of requirements, undermining the sense of progress and achievement.
 
-### The fallacy of testability
+### The Fallacy of Testability
 
-A primary motivator behind putting controls in Level 1 version 4.x was whether they could be checked using "black box" style external penetration testing which was not entirely in line with the concept of Level 1 being the minimum security controls. On the one hand, ASVS users would say that Level 1 was not sufficient for a secure application whilst on the other hand, users would complain that ASVS was too difficult to test.
+A key factor in selecting controls for Level 1 in version 4.x was their suitability for assessment through "black box" external penetration testing. However, this approach was not fully aligned with the intent of Level 1 as the minimum set of security controls. Some users argued that Level 1 was insufficient for securing applications, while others found it too difficult to test.
 
-Additionally, using testability as a factor is relative and in some cases misleading. Just because something is testable does not mean that it is testable in an automated or trivial way. Finally, the most testable requirements are not necessarily those that have the most important security impact or are the easiest to implement.
+Relying on testability as a criterion is both relative and, at times, misleading. The fact that a requirement is testable does not guarantee that it can be tested in an automated or straightforward manner. Moreover, the most easily testable requirements are not always those with the greatest security impact or the simplest to implement.
 
-### Not just about risk
+### Not Just About Risk
 
-The use of prescriptive, risk-based levels that mandate that a certain application has to be at a certain level seems overly opinionated in hind-sight. In reality, the order of implementing security controls will depend on factors including both risk reduction and also effort to implement.
+The use of prescriptive, risk-based levels that mandate a specific level for certain applications has proven to be overly rigid. In practice, the prioritization and implementation of security controls depend on multiple factors, including both risk reduction and the effort required for implementation.
```

Comment:
The same here - The section focuses what and why we don't use anymore, but maybe to point out here with one short line, what is used now in 5.0.